spec/lib/karafka/server_spec.rb

```diff
@@ -105,14 +105,12 @@
 
   describe '#start' do
     before do
-      allow(Karafka::App).to receive(:run!)
```

Comment:
Should we move this to the https://github.com/karafka/karafka/blob/master/spec/lib/karafka/runner_spec.rb?

Reply:
I do not see any reason of adding this there, it is fully covered by the integration suite and it's a leftover of old times when we didn't have them at all